src/etc/inc/interfaces.inc

```diff
@@ -2628,10 +2628,21 @@ function interface_configure($interface = 'wan', $reloadall = false, $linkupeven
             break;
     }
 
+	/* configure all ipv6 dhcp conf & scripts at this point  */
+    /* We need to take into account that a LAN change        */
+    /* may also affect it. It will get called everytime      */
+    /* interface is reconfigured for track or WAN dhcpv6,    */
+	/* but we have to watch all interfaces not just WAN.     */
+	/* Initialy this will only support one WAN interface     */
+	/* with dhcp6.            							     */
+	/* We call the function configure_dhcp6c_conf() when     */
+	/* either its a WAN dhcpv6 or a LAN track6 setting.      */
+
     if (isset($wancfg['ipaddrv6'])) {
         switch ($wancfg['ipaddrv6']) {
             case 'slaac':
             case 'dhcp6':
+				configure_dhcp6c_conf('wan', $config['interfaces']['wan']);
```

Comment:
Shouldn't this be $interface and $wancfg ?

Reply:
If you use $interface then I would  think it would try to set up dhcp6c conf using the LAN interface settings, as interface_configure is also called when the LAN interface is being configured, hence my forcing it to use WAN... or am I reading this wrong, It works as expected and re-writes the config on a change to LAN or WAN.

Reply:
Don‘t know yet. Statically assuming WAN is likely wrong. First review sweep for just asking questions. But this patch is manageable now, thanks again. We will figure this out together. 😊

Reply:
should be the same interface here, that's where it was split from :)